packages/core/admin/admin/src/pages/InternalErrorPage/tests/index.test.js

```diff
@@ -0,0 +1,462 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import { IntlProvider } from 'react-intl';
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+import InternalErrorPage from '../index';
+import { useModels } from '../../../hooks';
+
+jest.mock('../../../hooks', () => ({
```

Comment:
Why do you need to mock this hook since you don't import it in your component?

Reply:
That's a bad copy paste 🤣  Nice catch.